src/view.c

```diff
@@ -2296,6 +2296,16 @@ view_toggle_keybinds(struct view *view)
 	}
 }
 
+void
+view_warp_cursor_to_focused(struct view *view)
```

Comment:
`view_center_cursor` is probably a better name for this function, because its action has nothing to do with the view being focused.